diff --git a/app/src/libs/muparser/include/muParserDef.h b/app/src/libs/muparser/include/muParserDef.h
index 947c670..b189c6a 100644
--- a/app/src/libs/muparser/include/muParserDef.h
+++ b/app/src/libs/muparser/include/muParserDef.h
@@ -45,7 +45,7 @@
   This datatype must be a built in value type. You can not use custom classes.
   It should be working with all types except "int"!
 */
-#define MUP_BASETYPE double
+#define MUP_BASETYPE float
 
 /** \brief Activate this option in order to compile with OpenMP support.
 
diff --git a/app/src/libs/muparser/src/muParserTest.cpp b/app/src/libs/muparser/src/muParserTest.cpp
index d746107..547ebdd 100644
--- a/app/src/libs/muparser/src/muParserTest.cpp
+++ b/app/src/libs/muparser/src/muParserTest.cpp
@@ -1559,7 +1559,7 @@ namespace mu
 				p1->DefinePostfixOprt(_T("{m}"), Milli);
 				p1->DefinePostfixOprt(_T("{M}"), Mega);
 				p1->DefinePostfixOprt(_T("m"), Milli);
-				p1->DefinePostfixOprt(_T("n"), [](double value){return value * 1E-9;});
+				p1->DefinePostfixOprt(_T("n"), [](float value){return value * 1E-9f;});
 				p1->DefinePostfixOprt(_T("meg"), Mega);
 				p1->DefinePostfixOprt(_T("#"), times3);
 				p1->DefinePostfixOprt(_T("'"), sqr);
